src/import/sst_service.rs

```diff
@@ -656,7 +730,15 @@ macro_rules! impl_write {
                         }
                         _ => return Err(Error::InvalidChunk),
                     };
+                    // wait the region epoch on this TiKV to catch up with the epoch
+                    // in request, which comes from PD and represents the majority
+                    // peers' status.
```

Comment:
What if a peer is more up-to-date than the request (peer.region_epoch > request.region_epoch)?